docs/langchain.md

```diff
@@ -0,0 +1,96 @@
+# Deploying LangChain Agents on Fixie
+
+## Overview
+
+[LangChain](https://python.langchain.com/) is a popular framework for developing applications built on top of Large Language Models. Fixie supports LangChain as a first-class citizen. This means you can develop LangChain Agents/Chains/Prompts just as you typically would and then quickly deploy them onto Fixie infrastructure.
+
+## Why Deploy on Fixie?
+
+Deploying on Fixie gives you three major advantages:
+
+1. We host it for you (for free). Simply run `fixie deploy` and your Agent will be hosted in the cloud, complete with a REST API for calling it programatically and a shareable URL that will allow others to demo your agent in the browser.
+1. No need for an OpenAI key. Fixie will automatically take care of that, and supports 1,000 free requests per day, per user. So if your creation gets popular, you're not stuck footing the bill or asking people to mint their own key.
+1. It allows you to integrate with the rest of the Fixie ecosystem. This means other people in the community will be able to leverage your agent and vice-versa.
+
+## Getting Started
+
+First, you'll need to make sure you have `fixie` installed on your machine with `pip install fixieai`. This will install the Fixie CLI, which you'll need for deploying to Fixie. You'll also need a Fixie account and to authorize the Fixie CLI. You can do both by running `fixie auth`.
```

Comment:
I would use structure a little more rigorously here, so it's more scannable:

> ## Getting Started
> ### Command Line Setup
> 1. Install Fixie on your machine:
>     `$ pip install fixieai`
> 1. Create an account and link your command line to it:
>   `$ fixie auth`
> ### Deploy a LangChain project
> Let's imagine we have a simple [PromptTemplate]()
>     `<code snippet>`

Reply:
I agree this is tighter, but I don't like losing the "why". "Why do I need to install a CLI if I'm using LangChain". But let me take a pass at tightening.

Reply:
My comment was more about increasing structure than decreasing word count. I think we can retain the "why". I edited my original comment.